Makefile

```diff
@@ -0,0 +1,18 @@
+
+
+GIT_COMMIT     := $(shell git describe --dirty=-unsupported --always --tags || echo pre-commit)
+IMAGE_VERSION  ?= $(GIT_COMMIT)
+
+.PHONY: db-up
+db-up:
+	docker run --name pur-db -e POSTGRES_PASSWORD=infoblox123 -e POSTGRES_USER=postgres -e POSTGRES_DB=postgress -p 5432:5432 -d postgres:10.13
```

Comment:
I think it would be better to hide password

Reply:
this is a local setup only